libnetdata/storage_number/storage_number.c

```diff
@@ -81,23 +81,40 @@ storage_number pack_storage_number(calculated_number value, uint32_t flags) {
     return r;
 }
 
+// Lookup table to make storage number unpacking efficient.
+static calculated_number lut10x[4 * 8];
+
+__attribute__((constructor)) void initialize_lut(void) {
```

Comment:
I don't think there exists a compiler out there that manages to build netdata successfully without already supporting the constructor attribute. If you don't like this I can make this function visible in a header and call it from main.

Reply:
does this include MS compiler? we plan in future to move to Win

Reply:
`msvc` won't compile this code whereas `clang-cl` will.

Reply:
ok keep it as it is for now I guess. we will deal with this later